prefq/examples/imitation_preference_comparisons.py

```diff
@@ -131,9 +133,23 @@ def main():
         default=DEFAULT_SERVER_URL,
         help="Specify the server url (default: http://localhost:5000/)",
     )
+    parser.add_argument(
+        "--sshkey",
+        type=str,
+        default=None,
+        help="Specify SSH private-key filepath (default: None)",
```

Comment:
"Specify path to SSH private key used for authentication. Only necessary if the server is configured to require it using the corresponding public key."